src/pygeoprocessing/routing/routing.pyx

```diff
@@ -138,6 +138,18 @@ cdef struct FlowPixelType:
     int last_flow_dir
     double value
 
+cdef struct DecayingValue:
+    double decayed_value  # The value, which will be progressively updated as it decays
+    double min_value  # The minimum value before the Decaying Value should be ignored.
+

+cdef struct WeightedFlowPixelType:

```

Comment:
I think that this replaces ALL instances of `FlowPixelType`. Should we remove that struct definition?

Reply:
Interestingly, I'm still seeing `FlowPixelType` used in `flow_accumulation_mfd`, so I think it makes sense to leave it here for now.